templates/resource.nunjucks

```diff
@@ -59,12 +59,16 @@
 {% if (method.responses | length) != 0 %}
 <br/>
 <p><strong><em><big>Possible Responses</big></em></strong></p>
-<p>
-  {%- set comma = joiner() -%}
-  {%- for response in method.responses -%}
-  {{ comma() }} {{ response.code }}
-  {%- endfor -%}
-</p>
+<dl>
+{% for response in method.responses %}
+  <dt>{{ response.code }}</dt>
```

Comment:
To keep the layout consistent, we should maybe use the same for as for the other keys (as defined in `parameter-table.nunjucks`

```nunjucks
<p>
  <strong>{{ response.code }}</strong>
  {% markdown -%}
    {{ response.description }}
  {%- endmarkdown %}
</p>
```

You can massively screw up the layout with markdown here, but that's the user's own problem I suppose

Reply:
Alright, will fix that tomorrow. 

You are right, one could screw up the layout. But as you said, that's the user's own problem. We use it mainly to highlight params in `inline code` style and display line breaks.